cockroachdb/values.yaml

```diff
@@ -261,6 +273,17 @@ ingress:
   # - hosts: [cockroachlabs.com]
   #   secretName: cockroachlabs-tls
 
+# CockroachDB's metrics service
+metrics:
+  enabled: false
+  # CockroachDB's Prometheus operator ServiceMonitor support
```

Comment:
It might be nice to mention a bit more about what this is, either by expanding this comment or by adding a section to the README. Even just adding a link to Prometheus' documentation on ServiceMonitors would be helpful to people trying to understand what this section is about and whether they should use it.

Reply:
Mentioned in README